src/auth/auth.module.ts

```
@@ -0,0 +1,16 @@
+import { Module } from '@nestjs/common';
+import { JwtModule, JwtService } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { JwtAuthGuard } from './auth.guard';
+
+@Module({
+  imports: [
+    PassportModule,
+    JwtModule.register({
+      secret: process.env.JOSE_SECRET
```

ESLint

[error] Insert `,`

(prettier/prettier)